api/schemas/report.json

```diff
@@ -525,6 +525,11 @@
                     "type":        "boolean"
                 },
 
+                "failed_to_restart": {
+                    "description": "True if Puppet experienced an error while trying to restart this resource (Eg: notifying a Service resource), otherwise false.",
```

Comment:
nit: use "for example," instead of abbreviations

Reply:
I'll change this one, but it might be worth a ticket to clean up the other occurrences of things like that in the schema: https://github.com/puppetlabs/puppet/blob/eb92d95/api/schemas/report.json#L157